recipes/yajl/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, rm, rename
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.apple import fix_apple_shared_install_name
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class YAJLConan(ConanFile):
+    name = "yajl"
+    description = "A fast streaming JSON parsing library in C"
+    license = "ISC"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lloyd/yajl"
+    topics = ("json", "encoding", "decoding", "manipulation")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
```

Comment:
this is the default so we can skip it

```suggestion
        get(self, **self.conan_data["sources"][self.version], strip_root=True)
```